views/js/controller/runtime/testRunner.js

```diff
@@ -1216,13 +1216,18 @@ function (
         };
 
         var config = module.config();
+        var _isEmptyItemWarningModalEnabled = function() {
+            return config
+                && config.qtiTools
+                && config.qtiTools.emptyItemWarningModal;
+        };
```

Comment:
Is it a check for an external component? Not sure this is the better way. It introduces coupling, which is bad.
A more generic way should be possible.